lms/routes.py

```diff
@@ -24,14 +24,15 @@ def includeme(config):
         "/content_item_selection",
         factory="lms.resources.LTILaunchResource",
     )
+
     config.add_route(
-        "canvas_oauth_callback",
-        "/canvas_oauth_callback",
+        "canvas_api.oauth.authorize",
+        "/api/canvas/oauth/authorize",
         factory="lms.resources.OAuth2RedirectResource",
     )
     config.add_route(
-        "canvas_api.authorize",
-        "/api/canvas/authorize",
+        "canvas_api.oauth.callback",
+        "/canvas_oauth_callback",
```

Comment:
Unfortunately we can't easily fix this URL 'cos its been given out to Canvas instances

Reply:
I was immediately thinking about a migration path, but presumably this would involve adding two paths or a redirect and asking customers to update settings.

At which point a customer could quite legitimately ask: "What's in it for me?" "Nothing"